src/merkle/merkle-objects.ts

```diff
@@ -10,8 +11,13 @@ import { IpfsService } from '../services/ipfs-service.js'
 import { BloomFilter } from 'bloom-filters'
 import { StreamID } from '@ceramicnetwork/streamid'
 
+const packageJson = JSON.parse(
+  fs.readFileSync(new URL('../../package.json', import.meta.url), 'utf8')
+)
+
 const BLOOM_FILTER_TYPE = 'jsnpm_bloom-filters'
 const BLOOM_FILTER_FALSE_POSITIVE_RATE = 0.0001
+const BLOOM_FILTER_VERSION = packageJson['dependencies']['bloom-filters']
```

Comment:
nitpick: All CAPS variable names should be reserved for hard-coded constants, not anything that could be different at runtime across invocations.